docs/language/src/concepts/translations.md

```diff
@@ -25,7 +25,19 @@ The literal characters `{` and `}` may be included in a string by preceding them
 
 ## Plurals
 
-**TODO**
+A special kind of formatting is the plural formating, because the string may change if there is a single elemnts or several.
```

Comment:
```suggestion
A special kind of formatting is the plural formatting, because the string may change depending on if there is a single element or several.
```